services/core-web/src/components/noticeOfWork/applications/review/ReviewNOWApplication.js

```diff
@@ -82,208 +85,223 @@ export const ReviewNOWApplication = (props) => {
     return codeHash[value];
   };
 
-  const renderApplicationInfo = () => (
```

Comment:
Is this file mostly formatting changes?

Reply:
yep, except line 89-93 and passing some parameters to components